SR2019/2019-04-23-competition-wrap-up.md

```diff
@@ -0,0 +1,52 @@
+---
+to: Student Robotics 2019
+subject: Final competition details
+---
+
+![Competition Photo](https://studentrobotics.org/images/content/news/sr2019/sr2019-photo.jpg)
+
+Hi all,
+
+First off, we'd like to thank you for taking part in Student Robotics 2019 and making it such a successful event! An article about the competition, results, and game has been [posted on our website](https://studentrobotics.org/news/2019-04-09-sr2019-tlc-crowned-champion/)
+
+Congratulation again to our winners:
+
+|        Prize          |            Team
```

Comment:
Suggestion: I don't know how much control we have over the HTML in the sent emails, but it might be an idea to borrow the hover-text approach used in the news article so that hovering the prizes shows a description of what they were for?

I'm not sure this will work in all email clients though.

Reply:
I don't think there's quite that much control, i'm also about 99% sure it won't render correctly in email clients, those things are weird!